### ISSUE TITLE
Add in inequalities to the model

### ISSUE THREAD

**Issue by ChrisBeeley** [The-Strategy-Unit/nhp_model#96]
See attached paper

[Implementing the inequalities change factor in the NHP model 230724.docx](https://github.com/The-Strategy-Unit/nhp_model/files/13355052/Implementing.the.inequalities.change.factor.in.the.NHP.model.230724.docx)

- [x] Add inequalities to sample_params - use made-up HRG codes
- [x] Add made-up HRGs to synthetic data generation notebook
- [x] Parse inequalities in generate_run_params
- [x] Handle inequalities - in ActivityResampling? note it's IP _and_ OP

**Comment by matt-dray** [comment]
Hullo @yiwen-h, I see the related PR https://github.com/The-Strategy-Unit/nhp_model/pull/324 is blocked by inputs updates, which aren't being enacted until Sprint 6. Okay for this issue to go into Sprint 6 as well?

**Comment by yiwen-h** [comment]
yup sure thing. I imagine the PR will get really stale and need major refactoring
